ui/app/adapters/kv/metadata.js

```diff
@@ -69,7 +69,10 @@ export default class KvMetadataAdapter extends ApplicationAdapter {
   }
 
   deleteRecord(store, type, snapshot) {
-    const { backend, path } = snapshot.record;
-    return this.ajax(this._url(kvMetadataPath(backend, path)), 'DELETE');
+    const { backend, path, fullSecretPath } = snapshot.record;
```

Comment:
this adapter method isn't called from the list view, right? I thought destroy record was called on a secret data model in the list view

Reply:
When we chatted, I didn't think the correct model was being passed in on the list view. I was wrong—It was (the kv/metadata model was being passed in for each secret listed). So the adapter method is called from the list view and we don't have a duplicate DELETE on metadata being called on both adapters.

Reply:
oh nice!